docs/user-guide/creating-application/environment-overrides.md

```diff
@@ -25,24 +27,52 @@ Any changes in the configuration will not be added to the template, instead, it
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have [Admin role](../user-guide/global-configurations/authorization/user-access.md#role-based-access-levels) or above (along with access to the environment and applications) to change perform environment override.
+{% endhint %}
+
+Go to **App Configuration** → **Environment Overrides**. For each environment you can override the following configurations:
+* [Deployment Template](#deployment-template)
+* [ConfigMaps](#configmaps--secrets)
+* [Secrets](#configmaps--secrets)
 
-### Basic Configuration
+### Deployment Template
 
-The basic deployment configuration which you specified on the **Basic** GUI section will be visible for you to customize for your environment.
+{% hint style="info" %}
+Users who are not super-admins will land on [Basic (GUI)](#basic-gui) section when they visit the **Deployment Template** page; whereas super-admins will land on [Advanced (YAML)](#advanced-yaml) section. This is just a default behavior, they can still navigate to the other section if needed.
+{% endhint %}
 
-If you want to configure **Basic** GUI at the application level, then you can provide the required information in the [Basic Configuration](../creating-application/deployment-template/rollout-deployment.md#2.-basic-configuration).
+#### Basic (GUI)
 
-If `Basic` is locked, you can modify the configurations on `Advanced (YAML)` which will be the default page.
+If you have a [base deployment configuration](../creating-application/deployment-template/deployment.md#2.-basic-configuration) set up at application level, the environment(s) you define for your application will also inherit those values.
 
-{% hint style="warning" %}
-Super-admins can lock keys in deployment template to prevent non-super-admins from modifying those locked keys. Refer [Lock Deployment Configuration](../global-configurations/lock-deployment-config.md) to know more.
+However, you have the flexibility to use different values at the environment level by overriding the base configurations as shown below. 
+
+![Figure 2: Overriding Deployment Template - GUI Method](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/environment-overrides/base-config-override.gif)
+
+Refer [basic configuration](../creating-application/deployment-template/deployment.md#2.-basic-configuration) to know more about each field within `Basic (GUI)` section.
+
+{% hint style="info" %}
+### Want to customize the fields displayed on Basic (GUI)? [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
+Refer [Customize Basic GUI](../creating-application/deployment-template/deployment.md#customize-basic-gui) to know the process of adding, removing, and customizing the Basic (GUI) section of your Deployment Template.
```

Comment:
These steps are true in general, for devtron charts as well as custom chart, we need to make this common and not tie it with any specific chart type

Reply:
Made it common now.
Since this is **Environment Overrides** doc, the hintblock will continue to remain (as a CTA to customizing Basic GUI). But yes, it has been modified now to make it generic and not chart-specific.